### DISCUSSION TITLE
Working with multiple database schemas

### DISCUSSION THREAD

**Discussion by galkatz373** [prisma/prisma#3890] [Q&A] [answered]
We have a postgres database and I want to use prisma 2 for the backend, but we cannot seem to find a solution how to deal with multiple database schemas with prisma 2.
Maybe we missed something in the docs.
This is a must feature for our team, cause separating our tables into multiple schemas really helps organize the database.

Thanks in advance.

**Comment by ryands17** [comment, marked as answer]
Hey @galkatz373 👋 
Currently Prisma doesn't support splitting your models into multiple schemas, but there's a feature request https://github.com/prisma/prisma/issues/2377 that you can follow.

Do add a 👍 on this request so that we can prioritize this.

**Reply by galkatz373** [reply]
Thank you for the reply. You have an amazing team!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
